lib/active_merchant/billing/gateways/card_connect.rb

```diff
@@ -271,7 +295,11 @@ def message_from(response)
       end
 
       def authorization_from(response)
-        response['retref']
+        if response['profileid']
+          "#{response['acctid']}||#{response['profileid']}"
```

Comment:
Nitpick but this kind of makes it look like a concated split auth, when it isn't. Maybe pull the conditional outside of the interpolation string?

Reply:
Oh, hah, it is concating. In that case, the opposite of what I said; most separators are single character.

Reply:
Just to be clear, just do `|` instead of `||`?